tests/kotlin/KotlinLintBearTest.py

```diff
@@ -0,0 +1,39 @@
+import os
+import unittest
+from queue import Queue
+
+from coalib.settings.Section import Section
+from bears.kotlin.KotlinLintBear import KotlinLintBear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+
+
```

Comment:
What i meant was say you could do something like this here
```
bad_semicolon = """
package a.b.c;
    fun main() {
        fun name() { a(); return b }
        println(";")
        println();
}
"""
```
I think you can find some inspiration here
https://github.com/coala/coala-bears/blob/d658321228acd3a0317be5587a41d2e19f2bda40/tests/c_languages/CPPCleanBearTest.py

Reply:
This is not a global bear which is supposed to run on directories.